app/extensions/safe/blockNonSafeReqs.ts

```diff
@@ -17,43 +17,49 @@ const blockNonSAFERequests = () => {
 
     const safeSession = remote.session.fromPartition( CONFIG.SAFE_PARTITION );
 
-    safeSession.webRequest.onBeforeRequest( filter, ( details, callback ) => {
-    //  testcafe needs access to inject code
-        if ( isRunningTestCafeProcess ) {
-            callback( {} );
-            return;
-        }
+    safeSession.webRequest.onBeforeRequest(
+        filter,
+        ( details, callback ): void => {
+            //  testcafe needs access to inject code
+            if ( isRunningTestCafeProcess ) {
+                callback( {} );
+                return;
+            }
 
-        if ( urlIsAllowedBySafe( details.url ) ) {
-            logger.info( `Allowing url ${details.url}` );
-            callback( {} );
-            return;
-        }
+            if ( urlIsAllowedBySafe( details.url ) ) {
+                logger.info( `Allowing url ${details.url}` );
+                callback( {} );
+                return;
+            }
 
-        // HACK for idMgr and Patter. until:
-        // https://github.com/parcel-bundler/parcel/issues/1663
-        if ( details.url.includes( 'font_148784_v4ggb6wrjmkotj4i' ) ) {
-            const thePath = parseURL( details.url ).path;
-            const ext = path.extname( thePath );
+            // HACK for idMgr and Patter. until:
+            // https://github.com/parcel-bundler/parcel/issues/1663
+            if ( details.url.includes( 'font_148784_v4ggb6wrjmkotj4i' ) ) {
+                const thePath = parseURL( details.url ).path;
+                const extension = path.extname( thePath );
 
-            const newUrl = `http://localhost:${CONFIG.PORT}/dummy/iconfont${ext}`;
-            callback( { redirectURL: newUrl } );
-            return;
-        }
+                const newUrl = `http://localhost:${
+                    CONFIG.PORT
+                }/dummy/iconfont${extension}`;
+                callback( { redirectURL: newUrl } );
+                return;
+            }
 
-        if ( httpRegExp.test( details.url ) ) {
-            if ( allowedHttp.includes( details.url ) ) {
-                try {
-                    open( details.url );
-                } catch ( e ) {
-                    logger.error( e );
+            if ( httpRegExp.test( details.url ) ) {
+                if ( allowedHttp.includes( details.url ) ) {
+                    try {
+                        open( details.url );
+                        callback( { redirectURL: 'about:blank' } );
```

Comment:
Just needs `return` after this line so that `callback` is not called more than once.